### ISSUE TITLE
Page id: download form do not email, v1.0.16, (playground), dev (possible issue)

### ISSUE THREAD

**Issue by maevemacglinchey** [SuffolkLITLab/docassemble-CivilActionCoverSheet#96]
This seems a bit confusing - I think we should either show a preview or remove 'the following document has been created for you'. Any suggestions on how to make this simpler?
![image](https://user-images.githubusercontent.com/49881041/100582395-d7905680-32f1-11eb-8e99-e7d1793abd56.png)

**Comment by LGG233** [comment]
Good idea. Also, the phone number of the selected court doesn't come up - only the emergency help line. We need to fix that.
